### PR TITLE
Add delete dialog when deleting markdown blocks; fix cycling table bug; shift+enter open bundle detail in new window

### PR DESCRIPTION
**Pull request by yipenghe**
Screenshot from 2019-11-27 17-38-43

### CODE REVIEW

**Comment by yipenghe** [comment]
Also fixed the bug that if we press 'up' on the a table that has nothing above it, the focus cycles the table. Trivial fix.. didn't bother to open another PR for it

File: frontend/src/components/worksheets/Worksheet/Worksheet.js [hunk 1]
```diff
@@ -651,6 +662,10 @@ class Worksheet extends React.Component {
                             wsItems[focusIndex].mode === 'subworksheets_block')
                     ) {
                         // worksheet_item_interface and table_item_interface do the exact same thing anyway right now
+                        if (focusIndex === 0 && subFocusIndex === 0){
```
Comment (yipenghe) [inline frontend/src/components/worksheets/Worksheet/Worksheet.js]: This makes sure when the focus is on the very top, pressing up doesn't cycle around

File: frontend/src/components/worksheets/items/TableItem/BundleRow.js [hunk 2]
```diff
@@ -208,6 +208,14 @@ class BundleRow extends Component {
                 },
                 'keydown',
             );
+            Mousetrap.bind(
```
Comment (yipenghe) [inline frontend/src/components/worksheets/items/TableItem/BundleRow.js:211]: open bundle detail in new window

**Review by epicfaace — COMMENTED** [review]
Looks good -- can you rename `deleteItemCallBack` to `deleteItemCallback` though?

**Pull request merged**